### PR TITLE
ColModifierJsonArray helper

### PR DESCRIPTION
**Pull request by jamiemonksuk**
ColModifier helper to render out stored json array data in a presentable format.

It will also gracefully handle data that isn't JSON encoded.

Optionally pass through a constructor arg to not <pre> the output. This will enforce a HTML encode instead.

image

### CODE REVIEW

File: src/sprout/Helpers/ColModifierJsonArray.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+            // If not json, just return the value. No exploding plz.
+            if ($json_error !== JSON_ERROR_NONE or !is_array($json_array)) {
+                return Enc::html($val);
+            }
+        }
+
+        $out = '';
+        foreach ($json_array as $key => $data) {
+            $key = ucwords(str_replace('_', ' ', $key));
```
Comment (gwillz) [inline src/sprout/Helpers/ColModifierJsonArray.php:61]: I suppose this assumes that the body is snake_case. Would we want to also prettify a JSON body that uses camelCase?

I think `Infector::humanize()` would sort it out. But you'd still need the ucwords(). I wonder if we need an `Inflector::title()` helper for that.

Comment (jamiemonksuk) [reply src/sprout/Helpers/ColModifierJsonArray.php:61]: I like that title helper idea, I actually went looking for it in Inflector and Text first...

Comment (jamiemonksuk) [reply src/sprout/Helpers/ColModifierJsonArray.php:61]: There are camelcase helpers in Text already, just not to sentence case

Comment (jamiemonksuk) [reply src/sprout/Helpers/ColModifierJsonArray.php]: I did some stuffs like you suggested. It feels nice.

It's also very late though, and I'm quite tired, so please give it a good eyeballing :)

**Review by gwillz — APPROVED** [review]
Now that's some quality camels.

**Pull request merged**